assets/overlays/samba/generated/standalone/service.yaml

```diff
@@ -0,0 +1,31 @@
+# Generated file. Do not edit. Update using "make update".
+#
+# Loaded from base/controller_metrics_service.yaml
+# Applied strategic merge patch common/metrics/service_add_port.yaml
+# Applied strategic merge patch common/metrics/service_add_port.yaml
+#
+#
+
+apiVersion: v1
+kind: Service
+metadata:
+  annotations:
+    service.beta.openshift.io/serving-cert-secret-name: smb-csi-driver-controller-metrics-serving-cert
+  labels:
+    app: smb-csi-driver-controller-metrics
+  name: smb-csi-driver-controller-metrics
+  namespace: ${NAMESPACE}
+spec:
+  ports:
+  - name: provisioner-m
+    port: 9222
```

Comment:
This is 9223 in controller.yaml

Reply:
This is general problem - a bug. I'm fixing it in another commit:

`ad038fd8f80b111bee711ac1e813ec9d253d65ae` STOR-1726: fix port calculation in the generator

Surprisingly, our CI jobs have never caught it for aws-ebs / azure-disk / azure-file :(